src/notify/subscription/db.go

```diff
@@ -14,26 +17,18 @@ var prepStmts struct {
 func CreateDB(db *sql.DB) error {
 	myDB = db
 
-	_, err := myDB.Exec(`
-              CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
-              CREATE TABLE IF NOT EXISTS subscriptions (
-                id UUID PRIMARY KEY,
-                event_type VARCHAR(64),
-                context VARCHAR(64),
-                account_id UUID,
-                created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP )
-              `)
-	if err != nil {
+	if _, err := myDB.Exec(sqlSchema); err != nil {
 		return err
 	}
 
-	err = prepareStatements()
+	err := prepareStatements()
 
 	return err
 }
 
 func get(id string) (Subscription, error) {
 	var subscription Subscription
+
 	rows, err := prepStmts.get.Query(id)
```

Comment:
I didn't do this, but if you look at the example for QueryRow() https://golang.org/pkg/database/sql/#DB.QueryRow there is a way to handle "not found" differently if you need to.